include/hal/library/cryptlib/cryptlib_rng.h

```diff
@@ -27,4 +27,18 @@
  **/
 extern bool libspdm_random_bytes(uint8_t *output, size_t size);
 
+/**
+ * Generates a 64-bit random number.
+ *
+ * An implementation must be provided by the target environment to provide
+ * a cryptographically safe 64-bit random number.
+ *
+ * @param[out] rand_data     buffer pointer to store the 64-bit random value.
+ *
+ * @retval true         Random number generated successfully.
+ * @retval false        Failed to generate the random number.
+ *
+ **/
+extern bool libspdm_get_random_number_64(uint64_t *rand_data);
```

Comment:
This should be in https://github.com/DMTF/libspdm/blob/main/os_stub/spdm_crypt_ext_lib/cryptlib_ext.h.

Reply:
sorry about that! fixed.